programs/openbook-twap/src/lib.rs

```diff
@@ -567,6 +607,56 @@ pub mod openbook_twap {
         Ok(())
     }
 
+    pub fn prune_orders(ctx: Context<PruneOrders>, limit: u8) -> Result<()> {
+        let market_key = ctx.accounts.market.key();
+
+        let seeds =
+            TWAPMarket::get_twap_market_seeds(&market_key, &ctx.accounts.twap_market.pda_bump);
+        let signer_seeds = &[&seeds[..]];
```

Comment:
do you think worth adding the expire check here? or we can assume that openbook v2 handles this into perpetuity?

Reply:
(to be clear, I lean to not worth adding, just wanted to raise)

Reply:
Yep, we can rely on openbook to do the check here. I tested as well to make sure it fails manually. But can add an expected-to-fail test in the bankrun tests before the time jump happens.